src/client/app/components/about.ts

```diff
@@ -65,13 +100,34 @@ export class About implements AfterViewInit {
         this.showChange.emit(false);
     }
 
-    cache = `
-    ${Strings().aboutStorage}
-    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings().aboutSnippets)}
-    ${storageSize(sessionStorage, 'playground_intellisense', Strings().aboutIntellisense)}
-    `;
-
     setLanguage(languageCode: string) {
         this.currentChosenLanguage = languageCode;
     }
+
+    async changeConfig(newConfig: string) {
```

Comment:
Added this functionality. User now selects new environment and then after "okClicked" the user gets the confirmation message that I had earlier. Clicking "Cancel" will reset the dropdown to current environment.